### DISCUSSION TITLE
Supported Variables

### DISCUSSION THREAD

**Discussion by chanders** [OrcaSlicer/OrcaSlicer#245] [General]
Is there a list somewhere of variables that are supported in SoftFever? Thanks!

**Comment by hliebscher** [comment]
it's the same like BambuStudio 
i use the list from PrusaSlicer

current_extruder
current_object_idx
day
filament_preset
first_layer_print_convex_hull
first_layer_print_max
first_layer_print_min
first_layer_print_size
has_single_extruder_multi_material_priming
has_wipe_tower
hour
initial_extruder
initial_tool
input_filename
input_filename_base
minute
month
num_instances
num_objects
physical_printer_preset
print_bed_max
print_bed_min
print_bed_size
print_preset
printer_preset
scale
second
timestamp
total_layer_count
